website/pages/docs/[[...page]].jsx

```diff
@@ -1,33 +1,75 @@
 import { productName, productSlug } from 'data/metadata'
-import order from 'data/docs-navigation.js'
 import DocsPage from '@hashicorp/react-docs-page'
+import PluginTierLabel from 'components/plugin-tier-label'
+import DevAlert from 'components/dev-alert'
+// Imports below are only used server-side
 import {
   generateStaticPaths,
   generateStaticProps,
-} from '@hashicorp/react-docs-page/server'
+} from 'components/remote-plugin-docs/server'
 
-const subpath = 'docs'
+//  Configure the docs path
+const BASE_ROUTE = 'docs'
+const NAV_DATA = 'data/docs-nav-data.json'
+const CONTENT_DIR = 'content/docs'
+// override default "main" value for branch for "edit on this page"
+const MAIN_BRANCH = 'master'
+// add remote plugin docs loading
+const OPTIONS = {
+  remotePluginsFile: 'data/docs-remote-plugins.json',
+  additionalComponents: { PluginTierLabel },
+}
 
-export default function DocsLayout(props) {
+function DocsLayout({ isDevMissingRemotePlugins, ...props }) {
   return (
-    <DocsPage
-      product={{ name: productName, slug: productSlug }}
-      subpath={subpath}
-      order={order}
-      staticProps={props}
-      mainBranch="master"
-    />
+    <>
+      {isDevMissingRemotePlugins ? (
```

Comment:
This shows an alert bar in local dev previews for missing remote content ( if `GITHUB_API_TOKEN` isn't set ). This lets authors know that they're not seeing the full docs site picture, without forcing them to generate a token to use just to preview the site.